Structure_Engine/Query/DistributedPoints.cs

```diff
@@ -33,20 +37,32 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        public static List<Point> DistributedPoints(Bar bar, int divisions, double startLength = 0, double endLength = 0)
+        [Description("Gets a list of evenly distributed points along a bar from a given number of dividsions")]
+        [Input("bar", "The bar to get division points for")]
+        [Input("divisions", "Number of segments to divide the bar into. The number of points returned will be divisions +1.")]
+        [Input("startLength", "Optional offset from bar start node to start of divisions.", typeof(Length))]
+        [Input("startLength", "Optional offset from bar end node to start of divisions.", typeof(Length))]
```

Comment:
I feel like this one is meant to be `endLength`?

Reply:
you feel correctly!

Reply:
another one to check? That we do not have two inputs with the same name?

Reply:
Could add it to the list, are you ok raising an issue for it @IsakNaslundBh on Test_TK? :smile: